main.go

```diff
@@ -26,29 +27,38 @@ func main() {
 	ctx := context.Background()
 
 	if err := run(ctx); err != nil {
-		log.Event(ctx, "fatal runtime error", log.Error(err), log.FATAL)
+		log.Error(ctx, "fatal runtime error", err)
 		os.Exit(1)
 	}
 }
 
 func run(ctx context.Context) error {
 	signals := make(chan os.Signal, 1)
 	signal.Notify(signals, os.Interrupt, os.Kill)
-
-	// Run the service, providing an error channel for fatal errors
 	svcErrors := make(chan error, 1)
-	svcList := service.NewServiceList(&service.Init{})
-	svc, err := service.Run(ctx, svcList, BuildTime, GitCommit, Version, svcErrors)
+
+	// Read config
+	cfg, err := config.Get()
 	if err != nil {
-		return errors.Wrap(err, "running service failed")
+		log.Error(ctx, "unable to retrieve configuration", err)
+		return err
```

Comment:
Looks like we should be returning fmt.Errorf wrapped error here? Maybe I need to update on import-cantabular-dataset too

Reply:
Yes, please, I will change this one, if you could chagne cantabular-dataset that would be great